src/pretix/plugins/sendmail/templates/pretixplugins/sendmail/history.html

```diff
@@ -0,0 +1,51 @@
+{% extends "pretixcontrol/event/base.html" %}
+{% load i18n %}
+{% load bootstrap3 %}
+{% block title %}{% trans "Send out emails" %}{% endblock %}
+{% block content %}
+    <h1>{% trans "Email history" %}</h1>
+    <div>
+        <ul class="list-group">
+            {% for log in logs %}
+                <li class="list-group-item logentry">
+                    <p class="meta">
+                        <span class="fa fa-clock-o"></span> {{ log.datetime|date:"SHORT_DATETIME_FORMAT" }}
+                        {% if log.user %}
+                            <br/><span class="fa fa-user"></span> {{ log.user.get_full_name }}
+                        {% endif %}
+                        {% if log.display %}
+                            <br/><span class="fa fa-comment-o"></span> {{ log.display }}
+                        {% endif %}
+                    </p>
+                    <p>
+                        <strong>Email address:</strong> {{ log.parsed_data.recipients }}
```

Comment:
This is kinda pointless now when we display event-level logs. However, it would be nice to display the recipients in some other way, e.g. "Recipients: paid, expired and pending orders". This data is already in ``log.parsed_data``. Do you find a way to display it? If not, no worry, this might not be obvious and I can think of something if you need help.